### ISSUE TITLE
My Wacom Bamboo Pen & Touch doesn't work in my Lubuntu 22.10 (Ubuntu)  since some automatic OS updates ago  (the LED is just blinking)

### ISSUE THREAD

**Issue by jolofj** [linuxwacom/xf86-input-wacom#316]
My Wacom Bamboo Pen & Touch doesn't work in my Lubuntu 22.10 (Ubuntu)  since some automatic OS updates ago  (the LED is just blinking):

```shell
> lsb_release -a
``` 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.10
Release:        22.10
Codename:       kinetic

It worked fine before the latest OS updates and it works fine on my other old Lubuntu 16 computer.

I have tried to install latest drivers and I have tried to build from source code according to the instructions: 

https://github.com/linuxwacom/input-wacom/wiki/Installing-input-wacom-from-source
https://github.com/linuxwacom/xf86-input-wacom/wiki/Building-The-Driver

But that doesn't work.

I have tried many different USB jacks.

When I connect the wacom:

```shell
 sudo dmesg | grep error
``` 
[ 4574.829003] usb 3-2.2.3: device descriptor read/64, error -32
[ 4575.041013] usb 3-2.2.3: device descriptor read/64, error -32
[ 4576.220864] usb 3-2.2.3: device not accepting address 12, error -71

```shell
sudo dmesg | grep wacom
``` 
*empty result*

```shell
sudo dmesg | grep CTH-460
``` 
[ 4578.866194] usb 3-2.2.2: Product: CTH-460

```shell
lsusb
``` 
Bus 002 Device 004: ID 0d8c:0014 C-Media Electronics, Inc. Audio Adapter (Unitek Y-247A)
Bus 002 Device 003: ID 08bb:2902 Texas Instruments PCM2902 Audio Codec
Bus 002 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 005: ID 0bc2:2322 Seagate RSS LLC SRD0NF1 Expansion Portable (STEA)
Bus 004 Device 006: ID 1058:2620 Western Digital Technologies, Inc. Elements 2620
Bus 004 Device 004: ID 05e3:0626 Genesys Logic, Inc. Hub
Bus 004 Device 003: ID 05e3:0626 Genesys Logic, Inc. Hub
Bus 004 Device 002: ID 05e3:0626 Genesys Logic, Inc. Hub
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 015: ID 14cd:121d Super Top Mass Storage Device
Bus 003 Device 014: ID 056a:00d1 Wacom Co., Ltd CTH-460 [Bamboo Pen & Touch]
Bus 003 Device 004: ID 05e3:0610 Genesys Logic, Inc. Hub
Bus 003 Device 008: ID 062a:4182 MosArt Semiconductor Corp. 2.4G Keyboard Mouse
Bus 003 Device 007: ID 0518:0001 EzKEY Corp. USB to PS2 Adaptor v1.09
Bus 003 Device 005: ID 1a40:0101 Terminus Technology Inc. Hub
Bus 003 Device 003: ID 05e3:0610 Genesys Logic, Inc. Hub
Bus 003 Device 002: ID 05e3:0610 Genesys Logic, Inc. Hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

I find this info in usbview (the X-win software):

CTH-460
Manufacturer: Wacom Co.,Ltd.
Speed: 12Mb/s (full)
Bus:   3
Address:  14
USB Version:  2.00
Device Class: 00(>ifc )
Device Subclass: 00
Device Protocol: 00
Maximum Default Endpoint Size: 64
Number of Configurations: 1
Vendor Id: 056a

**Comment by jolofj** [comment]
The text was truncated. 

The rest of the text:


CTH-460
Manufacturer: Wacom Co.,Ltd.
Speed: 12Mb/s (full)
Bus:   3
Address:  14
USB Version:  2.00
Device Class: 00(>ifc )
Device Subclass: 00
Device Protocol: 00
Maximum Default Endpoint Size: 64
Number of Configurations: 1
Vendor Id: 056a
Product Id: 00d1
Revision Number:  1.06

Config Number: 1
	Number of Interfaces: 2
	Attributes: 80
	MaxPower Needed:  98mA

	Interface Number: 0
		Name: 
		Alternate Number: 0
		Class: 03(HID  ) 
		Sub Class: 01
		Protocol: 02
		Number of Endpoints: 1

			Endpoint Address: 81
			Direction: in
			Attribute: 3
			Type: Int.
			Max Packet Size: 9
			Interval: 4ms

	Interface Number: 1
		Name: 
		Alternate Number: 0
		Class: 03(HID  ) 
		Sub Class: 00
		Protocol: 00
		Number of Endpoints: 1

			Endpoint Address: 82
			Direction: in
			Attribute: 3
			Type: Int.
			Max Packet Size: 64
			Interval: 4ms








> lsmod | grep usb
snd_usb_audio         376832  7
snd_usbmidi_lib        45056  1 snd_usb_audio
snd_hwdep              20480  2 snd_usb_audio,snd_hda_codec
mc                     65536  1 snd_usb_audio
snd_pcm               159744  6 snd_hda_codec_hdmi,snd_hda_intel,snd_usb_audio,snd_hda_codec,snd_hda_core
snd_rawmidi            45056  2 snd_seq_midi,snd_usbmidi_lib
snd                   114688  35 snd_hda_codec_generic,snd_seq,snd_seq_device,snd_hda_codec_hdmi,snd_hwdep,snd_hda_intel,snd_usb_audio,snd_usbmidi_lib,snd_hda_codec,snd_hda_codec_realtek,snd_timer,snd_pcm,snd_rawmidi
usb_storage            77824  2 uas
usbhid                 65536  0
hid                   159744  3 hid_cmedia,usbhid,hid_generic

**Comment by jolofj** [comment]
It stopped working without me doing any change. I didn't change jack that the cable was connected to etc.

**Comment by jolofj** [comment]
Solved!

I found out that it was the software usb-guard that blocked my Wacom device. 

I naively installed  usb-guard  on random believing naively that it only interfered with USB-storage, but it seems to randomly block all kinds of USB devices.  Suddenly it started to block my USB keyboard and US mouse but it didn't block the USB-harddrive... for some reason .  

If you want to install usb-guard, I really recommend to read the instructions thouroughly first.

I you do 
```shell
sudo dmesg | grep usb
``` 
and it responds with   "Device is not authorized for usage" messages you might have usb-guard installed.
